src/main/kotlin/com/jetbrains/snakecharm/lang/parser/SnakemakeStatementParsing.kt

```diff
@@ -298,22 +298,6 @@ class SnakemakeStatementParsing(
         return result
     }
 
-    override fun filter(
```

Comment:
Ok, what about `SnakemakeParsingScope.kt`, it still uses `scope.inNoSmkKeywordsAllowed` and mentions #130 workaround. Also please revise `inParamArgsList` and `inRuleLikeSectionsList` methods of SnakemakeParsingScope.